.travis.yml

```diff
@@ -1,13 +1,11 @@
 language: node_js
 
 node_js:
-  - '8'
-  - '7'
-  - '6'
-
-before_install:
```

Comment:
Why did you delete these lines?

Reply:
Travis automatically [uses the latest version of each major we define](https://travis-ci.org/i18next/i18next-parser/jobs/636452976#L181) and [logs it](https://travis-ci.org/i18next/i18next-parser/jobs/636452976#L204) too

Reply:
@karellm Travis automatically [uses the latest version of each major we define](https://travis-ci.org/i18next/i18next-parser/jobs/636452976#L181) and [logs it](https://travis-ci.org/i18next/i18next-parser/jobs/636452976#L204) too